pandas_datareader/moex.py

```diff
@@ -3,7 +3,7 @@
 import pandas as pd
 
 from pandas_datareader.base import _DailyBaseReader
-from pandas_datareader.compat import StringIO, binary_type, concat, is_list_like
+from pandas_datareader.compat import StringIO, binary_type, concat, is_list_like, PY3
```

Comment:
PDR is python 3 only, so no need to import Python 3 or provide a path for Python 2.7.

Reply:
ok, deleted py27 backward compatibility.